libs/fgdbg/client/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+import org.jetbrains.compose.compose
+import org.jetbrains.compose.desktop.application.dsl.TargetFormat
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version "1.6.10"
+    id("org.jetbrains.compose") version "1.1.1"
+}
+
+group = "com.google.filament"
```

Comment:
should I suffix`.fgdbg` here 🤔

Reply:
No the group is correct. If we wanted to publish this we would differentiate it by using a separate artifact id containing `fgfbg`.